### ISSUE TITLE
how can I train on GPU?

### ISSUE THREAD

**Issue by insikk** [rdevon/DIM#28]
I tried below command, but it run training loop with cpu. Can you provide some examples?

```
CUDA_VISIBLE_DEVICES=3,4,5,6 python scripts/main.py local classifier --d.source CIFAR10 --encoder_config resnet19_32x32 --local.mode nce --t.epochs 1000 -d 3 -n gpu_test
```

**Comment by rdevon** [comment]
Even without the `CUDA_VISIBLE_DEVICES` it trains on CPU? It does GPU by default, so this is strange.

**Comment by insikk** [comment]
@rdevon sorry for bothering. I checked my pytorch version again, and it was cpu-only compiled version.  After changing it to gpu-support compiled version, it works.

**Comment by yuu-Wang** [comment]
> I checked my pytorch version again, and it was cpu-only compiled version. After changing it to gpu-support compiled version, it works.

Can you run out successfully? I urgently need your help. Thank you!
